plugins.d/Lets_Encrypt/dehydrated-wrapper

```diff
@@ -214,23 +218,21 @@ clean_finish() {
 [[ $(which authbind) ]] || fatal "Authbind not installed"
 
 for sig in INT TERM; do
-    trap "clean_finish 1
-    kill -$sig $$" $sig
+    trap 'clean_finish 1 kill -$sig $$' $sig
```

Comment:
It doesn't look like `clean_finish` takes more than one argument, this doesn't seem right to me.

Reply:
I'll double check, but I'm 99% sure that I did test it (with at least ctrl-c) and it seemed to work!?

I'll be back! :)